src/utils/event.js

```diff
@@ -1,5 +1,13 @@
 import dayjs from "dayjs";
 
+export const isEventPast = (date) => {
+  return date === null ? false : dayjs().isAfter(date, `MMM D`);
```

Comment:
Думаю здесь проще использовать short-circuit
```suggestion
  return date && dayjs().isAfter(date, `MMM D`);
```

Reply:
short-circuit это - название оператора && (и) ?

исправил

Reply:
Нет, сам оператор так и называют обычно "и". 
Short-circuit это именно код вида `a && b`, когда `b` не выполнится, если `a == false`, то есть произойдет "короткое замыкание" раньше, чем интерпретатор дойдет до выполнения\проверки b